requirements.txt

```diff
@@ -2,6 +2,11 @@ Jinja2==2.7.1
 MarkupSafe==0.18
 Pygments==1.6
 Sphinx==1.2.2
+blessings==1.6
+coveralls==0.5
 docutils==0.11
+flake8==2.4.1
 six==1.9.0
 sphinx-rtd-theme==0.1.6
+pytest==2.7.2
```

Comment:
maybe move these to `requirements-dev.txt` or something similar so that these are not installed when doing `pip install flake8-diff`?